node/rootwalker_test.go

```diff
@@ -47,7 +47,7 @@ func newClient() *Client {
 	r.Register("Thread", Thread{})
 	r.Register("Board", Board{})
 	r.Done()
-	c, e := NewClient(NewClientConfig(), skyobject.NewContainer(r))
+	c, e := NewClient(NewClientConfig(), r)
```

Comment:
This one.
`skyobject.NewContainer` Now require DB as a parameter and return `*container` (surprise)
But client wants registry.